### PR TITLE
Buffs dragon's explosive resistance towards devastating explosions. Light and heavy explosions unchanged vs dragon. 

### PR DESCRIPTION
**Pull request by Watermelon914**
## About The Pull Request

High devastation explosions always take 40% to 75% of a dragon's health away. Carp rifts can only ever take heavy explosive damage, making them more resistant maxcaps (previously, maxcaps would just instantly destroy them).

## Why It's Good For The Game

The #61531 PR raises a valid concern about dragon v maxcaps but their implementation is poor.

Dragons are countered hard by maxcaps. They have only two options when they see someone with a maxcap: Run or fight. If they run, the person with the maxcap will just destroy their portal and they are fucked either way. If they fight, they will die due to the fact that maxcaps will instantly kill a dragon. 
This change makes it so that a dragon will still take devastating damage from a maxcap but the two options stated above will be more viable. If they choose to fight, they will tank a large amount of damage and possibly die. If they run, the portal won't be immediately destroyed if the person with the maxcap blows up. The aim of this PR is to discourage usage of people suicide bombing the dragon by making it so that it doesn't instantly kill the dragon.

tl;dr discourages the use of maxcaps to fight dragons because maxcaps not fun, still makes them viable but not as easy.

## Changelog

:cl:
balance: Devastating explosions will now deal damage equivalent to 40% to 75% of a dragon's max health.
balance: Carp rifts will no longer be instantly destroyed to devastating explosions.
/:cl:

### CODE REVIEW

**Comment by adamkad1** [comment]
Great, i did not know how to do that, so its good to know someone did

**Comment by Watermelon914** [comment]
out of scope

**Comment by adamkad1** [comment]
> out of scope

Yeah. Maybe in some other PR

**Comment by techno0** [comment]
Doesnt the rift become indestructable after some time? Shoudnt the dragon then only be resistant to explosions while beeing near his rift or while its destructable? I would suggest only giving the dragon a resistance when its somewhat close to the rift.

**Comment by Watermelon914** [comment]
The resistance to explosions only applies to high devastation explosions, aka the ones that *instantly* kill the dragon. They will instead take a portion of the dragon's health

**Comment by adamkad1** [comment]
I know that dragon has a 5 minute timer to place first rift. I don't know if they have the same timer on second and third
